hotload.py

```diff
@@ -227,7 +256,7 @@ def main():
 
     conf = {
         "watch": [watchfiles],
-        "steps": [ClearTerminal(), reloaded_module],
+        "steps": [ClearTerminal(), ReloadModules(reloaded_module.module), reloaded_module],
```

Comment:
`ReloadModules(reloaded_module.module)` reads kind of funny to me.

Reply:
Yes, reading the implementation is necessary to understand this line.

Naming the argument could help: `ReloadModules(main=reloaded_module.module)` or `ReloadModules(exclude=reloaded_module.module)`.

But I am of course open to renaming `ReloadModules` and `reloaded_module` too.

I have not perfected names and such in this PR, I wanted to initially share the concept.

Reply:
We can polish later :)